src/sentry/static/sentry/app/views/settings/incidentRules/list.tsx

```diff
@@ -90,17 +98,38 @@ class IncidentRulesList extends AsyncView<Props, State> {
                   <RuleLink to={`/settings/${orgId}/incident-rules/${rule.id}/`}>
                     {rule.name}
                   </RuleLink>
-                  <Confirm
-                    priority="danger"
-                    onConfirm={() => this.handleRemoveRule(rule)}
-                    message={t('Are you sure you want to remove this rule?')}
-                  >
-                    <RemoveButton
-                      size="small"
-                      icon="icon-trash"
-                      label={t('Remove Rule')}
-                    />
-                  </Confirm>
+
+                  <MetricName>{getMetricDisplayName(rule.thresholdType)}</MetricName>
+
+                  <ThresholdColumn>
+                    <Thresholds>
+                      {rule.triggers.map(trigger => trigger.alertThreshold).join(', ')}
```

Comment:
With multiple triggers how does the reader know which threshold is for which trigger?

Reply:
Each trigger only has a set of 1 alert/resolution threshold, but has multiple actions (e.g. email, slack).